shared/applicability/package.yml

```diff
@@ -1,11 +1,69 @@
 name: "cpe:/a:{arg}:{ver_specs_cpe}"
 title: "Package {pkgname} is installed"
-bash_conditional: {{{ bash_pkg_conditional("{pkgname}") }}}
+bash_conditional: {{{ bash_pkg_conditional("{pkgname}", escape=True) }}}
 ansible_conditional: {{{ ansible_pkg_conditional("{pkgname}") }}}
 versioned: true
 template:
   name: platform_package
 args:
+  audit:
+    {{% if 'ubuntu' in product or 'debian' in product %}}
```

Comment:
Why are you deciding based on product when you decide based on packaging system in similar cases below?

Reply:
I don't remember why it's done this way it might come from some file that is removed by this PR. Nevertheless, I have changed this in the new commit below to be consistent with the other conditions in this file.